.github/workflows/auto-release.yml

```
@@ -0,0 +1,262 @@
+          # Use a temporary file to avoid shell escaping issues
+          echo '${{ steps.changelog_entry.outputs.CHANGELOG_JSON }}' > /tmp/changelog.json
+          
+          curl -X POST "$API_URL" \
+            -H "Content-Type: application/json" \
+            -H "Authorization: Bearer $CHANGELOG_SYNC_TOKEN" \
+            -d @/tmp/changelog.json \
+            || echo "⚠️ Database sync failed (non-critical)"
+      
+      - name: Create GitHub Release
+        if: steps.commits.outputs.SKIP_RELEASE != 'true'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)